presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainOrcPredicate.java

```diff
@@ -185,8 +187,14 @@ public boolean matches(long numberOfRows, Map<Integer, ColumnStatistics> statist
             }
         }
 
+        // did we read any bloom filter?
+        if (numberOfBloomFiltersChecked < 1) {
```

Comment:
Wouldn't the check be equivalent to:
```
(
  (not exists `column` such as `effectivePredicateDomains` contains `column`)
  OR
  (for each `column` such as `effectivePredicateDomains` contains that `column`, corresponding `effectivePredicateDomains`.isAll() AND `stripDomains` contains that `column`)
)
```

But well, maybe it's easier and faster to use simple counter. :P